packages/@internationalized/date/docs/index.mdx

```diff
@@ -0,0 +1,69 @@
+{/* Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. */}
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import {HeaderInfo, FunctionAPI, ClassAPI, TypeContext, InterfaceType, TypeLink, PageDescription} from '@react-spectrum/docs';
+import packageData from '@internationalized/date/package.json';
+
+---
+category: Date and Time
+keywords: [date, time]
+order: -1
+title: Introduction
+after_version: 3.0.0
+---
+
+# @internationalized/date
+
+The `@internationalized/date` package provides objects and functions for representing and manipulating dates and times in a locale aware manner.
```

Comment:
```suggestion
The `@internationalized/date` package provides objects and functions for representing and manipulating dates and times in a locale-aware manner.
```

Reply:
There is probably some of these in a bunch of places, might not be a big enough deal to change though.